src/fs.rs

```diff
@@ -43,6 +44,24 @@ pub fn archive_tarball(root: &Path, target: &Path, threads: u32) -> Result<()> {
     Ok(())
 }
 
+/// Make a squashfs (xz compressed)
+pub fn archive_squashfs(root: &Path, target: &Path, threads: u32) -> Result<()> {
+    let output = Command::new("mksquashfs")
+        .arg(root)
+        .arg(target)
+        .arg("-comp")
+        .arg("xz")
+        .arg("-processors")
+        .arg(format!("{}", threads))
+        .spawn()?
+        .wait_with_output()?;
```

Comment:
```suggestion
        .wait()?;
```

I would suggest not to hide the output of `mksquashfs` so that when an error occurs, the user will be able to tell what's wrong.